_concepts/agency-summary.md

```diff
@@ -11,32 +11,50 @@ category: "Summary Pages:"
 headline: Agency Summary
 
 copy:
-  - text: Below is an idea on how to display data for a specific federal agency. Please take a look and give us your feedback in the discussion section at the bottom of each tab.
+  - text: Below are two idea on how to display data by federal agency. Please take a look and give us your feedback in the discussion section at the bottom of each tab.
 
 discussion_title: "Discussion suggestions:"
 
 discussion_bullet_points:
-- Does the layout and information help you complete your tasks?
-- Is there anything that should be removed from the layout?
+- Would you like additional information or data? 
+- Would you like the data displayed in a different way?
 - Are the labels and terminology appropriate?
-- Are the groupings of data sensible?
-- What do you like and dislike?
+- Are the filters helpful?
 
 tabs_required: yes
 
 tabs:
-  - title: Version 0.1
-    url_hash: "#!version-0.1"
+  - title: Version 0.1A
+    url_hash: "#!version-0.1A"
     archived_text:
     description: "Key Features:"
     features:
-      - lorum ipsum
-    disclaimer_text: <a class="usa-cta external-link" hr' target="_blank">View an interactive version of the below image</a>
-    image: " "
-    image_alt_text: This image displays 
+      - Agency's mission
+      - Breakdown of spending from budget to programs and recipients
+      - Spending by geography
+      - Table of award transactions by fiscal year
```

Comment:
Lines 32-35 each need to start with `discussion`, so the line 32 would look like
```      - discussion: Agency's mission```